lib/serverengine/daemon.rb

```diff
@@ -47,14 +47,23 @@ def initialize(server_module, worker_module, load_config_proc={}, &block)
       @chuser = @config[:chuser]
       @chgroup = @config[:chgroup]
       @chumask = @config[:chumask]
+
+      @pid = nil
+      @command_pipe = @config.fetch(:command_pipe, nil)
```

Comment:
what does this option do...? This option is doing nothing, if my understanding is correct.